channels/iq.m3u

```diff
@@ -77,6 +77,8 @@ https://live.imamhossaintv.com/live/ih4.m3u8
 https://ur.imamhusseintv.com/live/ih4.m3u8
 #EXTINF:-1 tvg-id="IshtarTV.iq" tvg-name="Ishtar TV" tvg-country="IQ" tvg-language="Arabic" tvg-logo="http://live.ishtartv.com/files/ishtar_animated.gif" group-title="General",Ishtar TV (1080p)
 http://176.9.102.245:1935/iShtarTV/liveStream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Karbala" tvg-country="IQ" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Arabic",Karbala TV
```

Comment:
it's either Karbala or Karbala TV :)

Reply:
Thanks Fixed. I will let the overnight script generate the tvg values